app/services/UserService.py

```diff
@@ -90,5 +102,8 @@ def VisitHistory(self, username, filters):
             data = cursor.fetchall()
```

Comment:
Query is returning Nothing if the data equals the date

Reply:
fixed in commit [f5c7c70](https://github.com/49ers-DB/Atlanta-Movie/pull/29/commits/f5c7c70f19db0015eda1a6cbbd6e88568480951c)